test/benchmarks/Dockerfile

```diff
@@ -29,5 +27,7 @@ RUN mkdir benchmark_results
 RUN pip install protobuf==3.20.1 requests psutil google-cloud-bigquery torch
 RUN cd test/benchmarks && protoc --python_out=. profile.proto
 
-## start daemon and benchmark 
-CMD ["sh", "-c", "./cedana daemon start && python3 test/benchmarks/benchmark.py"]
\ No newline at end of file
+
+RUN go build && ./cedana bootstrap 
+
+CMD ["sh", "-c", "sudo -E ./cedana daemon start & sudo python3 test/benchmarks/benchmark.py"]
```

Comment:
This still has cedana daemon start ?

Reply:
yep - you can start it manually like this; it in effect just runs the rpc server. Take a look at the build-start-daemon file as well or client-daemon.go to see what I mean